### PR TITLE
Load projects etc in Map editor and Scripting page

### PR DESCRIPTION
**Pull request by hungnguyen94**
This loads the project, scripts etc in the Map and Scripting page. Adding an entity to the map/timeline will now pop up a modal with fewer options. The user won't need to specify which Project/Map the entity belongs to, because it will just use the current one.

### CODE REVIEW

File: src/main/webapp/app/scripting/scripting-new-dialog.html [hunk 1]
```diff
@@ -22,3 +44,42 @@ <h4 class="modal-title" id="myProjectLabel">Add a new cue</h4>
         </button>
     </div>
 </form>
+
+
+
+
+<!--<form name="editForm" role="form" novalidate ng-submit="vm.save()">-->
```
Comment (rubenwiersma) [inline src/main/webapp/app/scripting/scripting-new-dialog.html]: All commented, should be removed

**Comment by rubenwiersma** [comment]
Looks good. Have you also fixed the problem with groups not loading correctly in the timeline?

**Comment by hungnguyen94** [comment]
Not yet, but since that bug is also in `dev` and not specific to this feature, I though I'd make a new branch for that.

File: src/main/webapp/app/scripting/scripting-new-dialog.html [hunk 2]
```diff
@@ -1,16 +1,38 @@
...
+        </div>
+        <div class="form-group label-floating">
+            <label class="control-label" for="field_duration">Duration</label>
+            <input type="number" class="form-control" name="duration" id="field_duration" ng-model="vm.cue.duration"/>
+            <p class="help-block">Enter the duration</p>
+        </div>
+        <div class="form-group label-floating">
+            <label class="control-label" for="field_action">Action</label>
+            <input type="text" class="form-control" name="action" id="field_action" ng-model="vm.cue.action"/>
+            <p class="help-block">Wubba lubba dub dub</p>
```
Comment (rubenwiersma) [inline src/main/webapp/app/scripting/scripting-new-dialog.html]: Peace amongst worlds!

**Comment by hungnguyen94** [comment]
I removed the commented HTML.

**Pull request merged**